server/src/actors/project.rs

```diff
@@ -631,21 +655,26 @@ impl RetryAfter {
     }
 }
 
-impl Message for RetryAfter {
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+pub struct RateLimited(pub RetryAfterScope, pub RetryAfter);
```

Comment:
Can we name this `RateLimit`? Two reasons:

 1. It's a message, in which case an active name is more appropriate.
 2. It represents the actual rate limit.